### PR TITLE
Changed to string formatting and spacing around `=` operator

### PR DESCRIPTION
**Pull request by gitumarkk**
- Changed string formatting in views.py and form.py from using `+` to `%s`
- Removed spacing around `=` in registration.html and contact.html

### CODE REVIEW

File: umonya/umonya/apps/main/views.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def send_email_f(f):
     subject = "User Registration"
     message = ''
     for item in f.cleaned_data:
-        message = message + item.upper() + "\n" + str(f.cleaned_data[item]) + "\n\n"
+        message = "%s %s \n %s \n\n" % (message, item.upper(), str(f.cleaned_data[item]))
```
Comment (vhata) [inline umonya/umonya/apps/main/views.py]: Try not to leave too many spaces floating around.  It's not harmful, just a little messy.  You could remove the spaces around the newline here, for example.  Don't bother for now, I'm just saying, bear it in mind in future.

(I'm starting to sound like such a whitespace nazi.)

**Comment by vhata** [comment]
Again, it would be preferable to do this in two commits - one to clean up the spaces, and the other to use string formatting instead of `+`.  But anyway, this looks good to me. :+1:

File: umonya/umonya/apps/main/views.py (hunk 1 above)
Comment (gitumarkk) [inline umonya/umonya/apps/main/views.py]: hahaha cool, will bear it in mind

**Comment by gitumarkk** [comment]
Okay will do that, but I thought that it is better to do an almost complete commit initially, but thanks for the insight.

**Comment by vhata** [comment]
It is better to do a complete commit, in that you should avoid committing half a solution.  But you should also avoid committing two solutions in one.

The reason for this is that when somebody is reviewing your code, they might want to look at it a single commit at a time, to see that each one does what it is supposed to do (or fixes the bug it claims it fix), without having to see all the other changes at the same time.

A good example would be fixing PEP8 whitespace at the same time as fixing a very hairy logic bug.  If I am trying to understand the new logic, it would be very distracting to see dozens of other lines all being modified as well. I might wonder why those lines are relevant to the bug, and I'd have to examine each one to see whether it was just adding line-breaks and indenting, or whether the code actually changes.

I hope that makes sense.  I don't want to bog you down with petty gripes about whitespace and the size of your commits, but if you gradually learn to do this stuff by habit, it makes you a much better coder in the long run. :)

**Pull request closed without merging**